helper-cli/src/main/kotlin/commands/packageconfig/CreateCommand.kt

```diff
@@ -174,8 +182,15 @@ internal class CreateCommand : CliktCommand(
         }
 
     private fun getNonOffendingLicenses(): Set<SpdxSingleLicenseExpression> {
+        val result = mutableSetOf<SpdxSingleLicenseExpression>()
+
         // Filter blanks to allow passing an empty list as argument to simplify caller code.
-        if (nonOffendingLicenseCategories.all { it.isBlank() }) return emptySet()
+        nonOffendingLicenseIds.filterNot { it.isBlank() }.mapTo(result) {
+            SpdxSingleLicenseExpression.parse(it)
+        }
+
+        // Filter blanks to allow passing an empty list as argument to simplify caller code.
+        if (nonOffendingLicenseCategories.all { it.isBlank() }) return result
```

Comment:
I know the code was like that before, but strictly speaking this code does not *filter*, but return early if *all* `nonOffendingLicenseCategories` are blank. Maybe this is a good opportunity to fix this? Isn't `nonOffendingLicenseCategories.isEmpty()` sufficient to account for `--non-offending-license-categories=` being specified (which I believe to be the intention)?

Reply:
It allows for specifying `--non-offending-license-categories ""`

Reply:
For reference, I just checked, and *clikt* returns a list with a single element which is an empty string in this case.

Reply:
Yes, this was the reason the code was invented.